crates/re_types/src/archetypes/transform3d_ext.rs

```diff
@@ -0,0 +1,67 @@
+use crate::datatypes::{
+    Rotation3D, Scale3D, TranslationAndMat3x3, TranslationRotationScale3D, Vec3D,
+};
+
+use super::Transform3D;
+
+impl Transform3D {
+    /// Identity transform, i.e. parent & child are in the same space.
+    pub const IDENTITY: Self = Self {
+        transform: crate::components::Transform3D::IDENTITY,
+    };
+
```

Comment:
all these new helpers should be `#[inline]` I reckon. A few are and I don't see the pattern.

Here and in the component & datatype